assets/src/less/blocks/wp-block-newsletter/layout.less

```diff
@@ -0,0 +1,16 @@
+/**
+ * Newsletter Signup Form Block
+ */
+
+.wp-block-cata-newsletter__wrapper {
+	position: relative;
+}
+
+.wp-block-cata-newsletter__bg-img {
+	position: absolute;
+	z-index: -1;
+	width: 100%;
+	height: 100%;
+	object-fit: cover;
+	overflow: hidden;
```

Comment:
Does it need the `overflow:hidden` ? Usually that would go on the relative parent like `wp-block-cata-newsletter__wrapper` but since you're doing `width: 100%; height: 100%;` there shouldn't be any overflow.

Reply:
Good point, removed.